core/embed/bootloader/messages.c

```diff
@@ -410,8 +415,8 @@ static bool _read_payload(pb_istream_t *stream, const pb_field_t *field,
     // update loader but skip first block
     if (firmware_block > 0) {
       ui_screen_install_progress_upload(
-          250 + 750 * (firmware_block * IMAGE_CHUNK_SIZE + chunk_written) /
-                    (firmware_block * IMAGE_CHUNK_SIZE + firmware_remaining));
+          1000 * (firmware_block * IMAGE_CHUNK_SIZE + chunk_written) /
```

Comment:
Is this intentional?

Reply:
There is no longer an erase phase before writing the firmware into the flash memory, as there was in the past. So yes, this change is intentional.